OCPBUGS-15980: small OTA typo (#63563)
openshift/openshift-docs: modules/update-common-terms.adoc

ifndef::openshift-origin[]

> Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that will cause fewer reboots of worker nodes.

Reviewer: ```suggestion
Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that results in fewer reboots of worker nodes.
```

Reviewer: Change looks good and one other thing to avoid future tense if possible, according to IBMSG.
